src/index.ts

```diff
@@ -40,6 +40,52 @@ const createGenerator: CodegenGeneratorConstructor = (config, context) => {
 			return !!value.mediaType.mimeType.match('\\bjson$')
 		})
 
+		hbs.registerHelper('hasOperationSupportingMultipart', function(value: CodegenOperation | CodegenOperation[]): boolean {
+			if (isCodegenOperation(value)) {
```

Comment:
Let's use the `containsMultipartOperation` function here too. It has the logic we need.

e.g.

```typescript
return containsMultipartOperation([value])
```